### ISSUE TITLE
peers parameter spec

### ISSUE THREAD

**Issue by jfromaniello** [hij1nx/level-replicator#23]
If I don't want to use multicast mode, what should I pass on the `peers` parameter. It looks like an array from the readme, but it seems to work with a hash.

Thanks

**Comment by heapwolf** [comment]
You can create a start list of known servers, possibly in concert with a service registry.

**Comment by jfromaniello** [comment]
Yes, the quedtion was more about the format, it is not an array of string
but a hash... and it seems to take the server address from the keys
